include/GLFW/glfw3.h

```diff
@@ -1719,6 +1719,31 @@ GLFWAPI GLFWmonitor* glfwGetPrimaryMonitor(void);
  */
 GLFWAPI void glfwGetMonitorPos(GLFWmonitor* monitor, int* xpos, int* ypos);
 
+/*! @brief Returns the work area of the monitor.
+ *
+ *  This function returns the position, in screen coordinates, of the upper-left
+ *  corner of the specified monitor.
+ *
+ *  Any or all of the position arguments may be `NULL`.  If an error occurs, all
```

Comment:
The length arguments can also be NULL, according to the implementation.

Reply:
Fixed this here: https://github.com/dougbinks/glfw/commit/875cbdf46d12e4d23d41e80069cd921f587e5829